packages/artifacts.txt

```diff
@@ -354,280 +352,340 @@ lib/js/sys.js
 lib/js/uchar.js
 lib/minisocket.js
 lib/ocaml/arg.cmi
+lib/ocaml/arg.cmj
```

Comment:
Any other consequences to the created package?
I guess you've tested installing the package.

Reply:
No other consequences. I have tested the package.